common/eth2/src/lighthouse/attestation_rewards.rs

```diff
@@ -0,0 +1,41 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards paid for attestations
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub struct IdealAttestationRewards {
+    // Validator's effective balance in gwei
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
+    pub effective_balance: u64,
+    // Ideal attester's reward for head vote in gwei
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
+    pub head: u64,
+    // Ideal attester's reward for target vote in gwei
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
+    pub target: u64,
+    // Ideal attester's reward for source vote in gwei
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
+    pub source: u64,
+}
+
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub struct TotalAttestationRewards {
+    // one entry for every validator based on their attestations in the epoch
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
+    pub validator_index: u64,
+    // attester's reward for head vote in gwei
+    pub head: i64,
```

Comment:
We know the head reward is non-negative so lets use a u64 for it.

```suggestion
    #[serde(with = "eth2_serde_utils::quoted_u64")]
    pub head: u64,
```